contracts/modules/utils/RequireUtils.sol

```diff
@@ -61,11 +77,100 @@ contract RequireUtils {
             )
           )
         )
-      ) == _wallet, "RequireUtils#requireConfig: UNEXPECTED_COUNTERFACTUAL_IMAGE_HASH");
+      ) == _wallet, "RequireUtils#publishConfig: UNEXPECTED_COUNTERFACTUAL_IMAGE_HASH");
     }
 
     // Emit event for easy config retrieval
     emit RequiredConfig(_wallet, imageHash, _threshold, abi.encode(_members));
+
+    if (_index) {
+      // Register last event for given wallet
+      lastWalletUpdate[_wallet] = block.number;
+    }
+  }
+
+  function publishSigners(
+    address _wallet,
+    bytes32 _hash,
+    uint256 _sizeMembers,
+    bytes memory _signature,
+    bool _index
+  ) external {
+    // Decode and index signature
+    (
+      uint16 threshold,  // required threshold signature
+      uint256 rindex     // read index
+    ) = _signature.readFirstUint16();
+
+    bytes32 imageHash = bytes32(uint256(threshold));
+
+    Member[] memory members = new Member[](_sizeMembers);
+    uint256 membersIndex = 0;
+
+    while (rindex < _signature.length) {
+      // Read next item type and addrWeight
+      uint256 flag; uint256 addrWeight; address addr;
+      (flag, addrWeight, rindex) = _signature.readUint8Uint8(rindex);
+
+      if (flag == FLAG_ADDRESS) {
+        // Read plain address
+        (addr, rindex) = _signature.readAddress(rindex);
+      } else if (flag == FLAG_SIGNATURE) {
+        // Read single signature and recover signer
+        bytes memory signature;
+        (signature, rindex) = _signature.readBytes66(rindex);
+        addr = recoverSigner(_hash, signature);
+
+        // Required signer event
+        emit RequiredSigner(_wallet, addr);
+
+        if (_index) {
+          // Register last event for given signer
+          lastSignerUpdate[addr] = block.number;
+        }
+      } else {
```

Comment:
What if one of the initial signer is a sequence wallet? It seems like we should support ERC-1271 signatures here too.

Reply:
I didn't wanted to make this PR dependent on #105, I think it should be the other way around